app/src/main/java/com/codesoom/assignment/controllers/TaskController.java

```diff
@@ -0,0 +1,48 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.models.Task;
+import com.codesoom.assignment.service.TaskService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.PathVariable;
+
+import java.util.List;
+
+/**
+ * HTTP 프로토콜을 통한 Task Entity를 처리하는 컨트롤을 담당합니다.
```

Comment:
javadoc 주석은 평범한 주석이 아니라 외부에 공개하는 "코드"로 생각하고 작성하는 것이 요령입니다. 이 클래스에서 `Task` 엔티티를 사용한다는 것을 외부에 공개할 의무는 없죠. "할 일" 정도로 추상화한 단어를 사용하는 것이 더 낫습니다. 

```suggestion
 * HTTP 프로토콜을 통한 할 일의 처리를 담당합니다.
```

`할 일`이 너무 제너럴한 명사이고 굳이 `Task`를 사용하고 싶다면 다음과 같이 할 수도 있겠죠. 

```suggestion
 * Task에 대한 조회와 변경 요청을 처리합니다.
```